buildpack.md

```diff
@@ -639,6 +647,17 @@ In either case,
 | `CPATH`           | `/include`   | header files     | [x]   |
 | `PKG_CONFIG_PATH` | `/pkgconfig` | pc files         | [x]   |
 
+#### Buildpack Details
```

Comment:
I'm up for removing this now that each buildpack knows who it is.

Reply:
Maybe we just keep `BP_DIR`?

Reply:
We've been able to determine this to date without an environment variable, so I'd say continue with the fewest environment variables possible.

Reply:
Removing from this PR.